### ISSUE TITLE
Update minSdkVersion and targetSdkVersion in the variables.gradle file

### ISSUE THREAD

**Issue by marioshtika** [ionic-team/capacitor-configure#57]
Hello everyone.

I want to change the `minSdkVersion` and the `targetSdkVersion` in the `variables.gradle` file.

I couldn't find a way to do this neither on the `capacitor.config.ts` file nor using the `capacitor-configure` tool.

Am I missing something or is this not implemented yet?

Thank you in advance.

**Comment by mlynch** [comment]
Thanks, I started working on this today and I need to make some changes to the Gradle parser to support variable declarations like this. But once I do that this should work well.

**Comment by mlynch** [comment]
Okay this should be available now in `2.0.7@next`, can you give that a try?

You can use it like this in the configure tool:

```yaml
      - file: variables.gradle
        target:
          ext:
            minSdkVersion:
        replace:
          minSdkVersion: 30
```

Or, with the Project API:

```typescript
    const gradle = await project.android?.getGradleFile('variables.gradle');
    await gradle?.parse();

    await gradle?.replaceProperties({
      ext: {
        minSdkVersion: {
        }
      }
    }, { minSdkVersion: 42 });
```

Let me know if that works for you!

**Comment by mlynch** [comment]
This is out in 2.0.8, please give it a try and let me know if it works for you!
